src/utils/functions.js

```diff
@@ -112,7 +115,7 @@ const renderFunction = (func, index, funcName) => {
 
 	return (
 		<section className={css.function} key={index}>
-			<dt id={id}>{name}(<var>{paramStr}</var>){returnType ? <span className={css.returnType}><Type>{returnType}</Type></span> : null}</dt>
+			<DefTerm id={id}>{name}(<var>{paramStr}</var>){returnType ? <span className={css.returnType}><Type>{returnType}</Type></span> : null}</DefTerm>
```

Comment:
Can the FloatingAnchor be a standalone element that exists between the `<section>` and `<dt>` elements? If it's absolute positioned, it wouldn't interfere with the layout, and would be positioned at the same starting place of the `dt`.

Reply:
That's possible but I'm not clear how that improves on the implementation. What problem do you expect it will address?

Reply:
Ideally, we don't have elements that position their internal parts outside of themselves. That creates problems when working with layouts and alignment. If this new icon needs to be on every heading, the page layout needs to be updated to accommodate it.

An alternate implementation is to simply add the FloatingAnchor element inline, after the title/heading text. This would require no layout/positioning hackery, would preserve the alignment of the elements on the page, and shouldn't be hard to implement, while still providing a recognizable hover-link.

Reply:
That's a fair point but is inconsistent with the rest of the docs which float the link to the left. I can investigate how to adjust that.

Reply:
Those other places should likely be updated too.
![Kapture 2019-06-26 at 15 21 02](https://user-images.githubusercontent.com/386529/60219055-13cea600-9826-11e9-9474-a7c30d706e97.gif)

Reply:
Noting also that there are places where that isn't possible such as properties for which the text is right aligned so the anchor placement would be inconsistent on those.

Reply:
I'd prefer to keep the GitHub style links, if possible.